### PR TITLE
Update Lava Puddles Map and Keep is disabled

### PR DESCRIPTION
**Pull request by moreavy**
screenshot

### CODE REVIEW

**Review by Panquesito7 — COMMENTED** [review]
The screenshot looks good, but you should probably disable HUD and chat, and any other texts.

**Comment by moreavy** [comment]
> The screenshot looks good, but you should probably disable HUD and chat, and any other texts.

Done!

File: lava_puddles/map.conf [hunk 1]
```diff
@@ -14,4 +14,4 @@ team.2.color = blue
 team.2.pos = (-51,6,-61)
 team.1 = red
 
-license = CC BY-SA 4.0 for mapwork. CC BY 3.0 for Skybox, adopted from TajamSoft:opengameart.org (https://opengameart.org/content/elyvisions-skyboxes)
+license = CC BY-SA 4.0 for mapwork. CC BY-SA 4.0 for screenshot by Elvis26. CC BY 3.0 for Skybox, adopted from TajamSoft:opengameart.org (https://opengameart.org/content/elyvisions-skyboxes)
```
Comment (JostP) [inline lava_puddles/map.conf:17]:
```suggestion
license = CC BY-SA 4.0 | Skybox textures adapted from the skybox "elyvisions skyboxes" (from https://opengameart.org/content/elyvisions-skyboxes) by elyvisions, originally licensed under CC BY 3.0.
```

**Review by JostP — APPROVED** [review]

**Comment by moreavy** [comment]
Would need to change screenshot.

File: lava_puddles/map.conf [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+
+team.2 = blue
+team.1.pos = (36,9,59)
+team.1.color = red
+team.2.color = blue
+team.2.pos = (-51,6,-61)
+team.1 = red
+
+license = CC BY-SA 4.0 | Skybox textures adapted from the skybox "elyvisions skyboxes" (from https://opengameart.org/content/elyvisions-skyboxes) by elyvisions, originally licensed under CC BY 3.0.
+hint = You may find diamonds hidden somewhere.
```
Comment (JostP) [inline lava_puddles/map.conf:18]:
```suggestion
hint = You can find diamonds hidden somewhere.
```

**Review by JostP — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
The map seems to be working fine, screenshot looks pretty cool I have to say. 🙂
However, the barrier in the top should probably be made higher, I only used 30 blocks of cobble to get to the top (not sure if it's a good idea though).

**Comment by moreavy** [comment]
> The map seems to be working fine, screenshot looks pretty cool I have to say. 🙂
> However, the barrier in the top should probably be made higher, I only used 30 blocks of cobble to get to the top (not sure if it's a good idea though).

@JostP is there anyway I can make the barrier higher?

**Comment by JostP** [comment]
> > The map seems to be working fine, screenshot looks pretty cool I have to say. slightly_smiling_face
> > However, the barrier in the top should probably be made higher, I only used 30 blocks of cobble to get to the top (not sure if it's a good idea though).
> 
> @JostP is there anyway I can make the barrier higher? Or would it be better keep the height same?

There is a way, but it is not necessary to do it. Nothing is wrong with having one map with low ceiling.

**Review by Panquesito7 — APPROVED** [review]
Can be merged, looks cool. 👍

**Review by MoreKat — COMMENTED** [review]
Nothing really unique to this map apart from the big amount of lava. It's just a plain and hilly area, like in many existing maps. The lava is also more of a burden than a blessing imo. You can easily cover most of the surface with lava by digging the blocks keeping it in place. I know you added dirt to initial items so the players can repair that, but players shouldn't have to worry about repairing the map they're playing in, to be able to play in it.
Honestly, I think you're relying too much on the skybox to make the map nice and unique, but the skyboxes are just for aesthetical purposes and don't add anything to the gameplay itself.

**Comment by moreavy** [comment]
I am closing this PR temporarily for a day. I'll reopen it soon.

**Comment by JostP** [comment]
@moreavy We with staff members decided that current version is unfortunatelly not enough unique to be added. You have made some huge steps to improve it, its smallness, not-plain andskybox are really good improvements. But as Kat said the skyboxes are just for aesthetical purposes and don't add anything to the gameplay itself.

So you wll have to use some more creativity to improve a bit current version. On the next map voting we will probably disable or request updates on maps below 3.0 average.

Current version is perfect to be improved. I suggest you to maybe add some small amount of buildings that are useful for gameplay (defense towers for example). And important, making lava more useful would definitely make map more unique and not similar to other maps. Useful means making it that players are able to use it for let's say defense or protection or whatever. Currently you can easily cover most of the surface with lava by digging the blocks keeping it in place. Thats not really good point of having lava on map I guess.
Let us know what you will include in an update to be able to advise or recommend you something

**Comment by moreavy** [comment]
> @moreavy We with staff members decided that current version is unfortunatelly not enough unique to be added. You have made some huge steps to improve it, its smallness, not-plain andskybox are really good improvements. But as Kat said, the skyboxes are just for aesthetical purposes and don't add anything to the gameplay itself.
> 
> So you wll have to use some more creativity to improve a bit current version. On the next map voting we will probably disable or request updates on maps below 3.0 average.
> 
> Current version is perfect to be improved. I suggest you to maybe add some small amount of buildings that are useful for gameplay (defense towers for example). And important, making lava more useful would definitely make map more unique and not similar to other maps. Useful means making it that players are able to use it for let's say defense or protection or whatever. Currently you can easily cover most of the surface with lava by digging the blocks keeping it in place. Thats not really good point of having lava on map I guess.
> Let us know what you will include in an update to be able to advise or recommend you something.
> 
> Sorry for late replay, there was a discussion on discord staff channel about this updated map.

@JostP I think its fine to merge this PR now.

**Comment by moreavy** [comment]
For the lava to not cover the land, I have a solution i.e. I'll need to create `ctf_map:permafrost` and `ctf_map:permafrost_with_stone`. I'll make the node surrounding the lava Indestructible and keep the lava at the base breakable for Base Defense using lava. I'll also add a few defense towers. I'll update the map on this weekend and create PR in https://www.github.com/MT-CTF/capturetheflag in the `ctf_map_core` mode to add Indestructible Permafrost and Indestructible Permafrost with stone. Please tell your opinion to do as so.

**Review by RyanZem — COMMENTED** [review]
This map looks nice, but if I will dig the puddle then the lava will flow to down and players won't be able to walk, but I can place dirt on the lava, so it's not a very big problem. Would be good if you will make around the puddles ctf_map:permafrost.

**Comment by moreavy** [comment]
I'll reopen when I finish the changes.

**Pull request closed without merging**